XMonad/Actions/PhysicalScreens.hs

```diff
@@ -21,6 +21,11 @@ module XMonad.Actions.PhysicalScreens (
                                       , sendToScreen
                                       , onNextNeighbour
                                       , onPrevNeighbour
+                                      , horizontalScreenOrderer
+                                      , ScreenComparator(ScreenComparator)
+                                      , getScreenIdAndRectangle
```

Comment:
I am not very proud to export this helper but I need it in  XMonad/Util/WorkspaceCompare.hs any suggestion how to avoid that ?